### PR TITLE
svg_loader: Add missing implementation of skewX and skewY in transform-list

### LINKED ISSUE #1927: transform-list attributes can't handle skewX and skewY
They are part of the transform-list specification: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#skewx

They seem to be able to handled well as part of matrices, but skewX and skewY seem to be treated like ThorVG doesn't know they exist.

skewX(n) should apply this transformation matrix (with n transformed from degrees to radians):
```
1 n 0
0 1 0
```

skewY(n) should apply this transformation matrix (with n transformed from degrees to radians):
```
1 0 0
n 1 0
```

### PR DESCRIPTION
**Pull request by MewPurPur**
I haven't figured out how to test this yet, and I don't know if changes are needed in more places. But this should be all the basic logic needed.

Fixes #1927

### CODE REVIEW

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 1]
```diff
@@ -786,6 +786,16 @@ static Matrix* _parseTransformationMatrix(const char* value)
             if (ptCount == 2) sy = points[1];
             Matrix tmp = { sx, 0, 0, 0, sy, 0, 0, 0, 1 };
             _matrixCompose(matrix, &tmp, matrix);
+        } else if (state == MatrixState::SkewX) {
+            if (ptCount != 1) goto error;
+            auto deg = (float)tan(points[0] * (M_PI / 180.0));
+            Matrix tmp = { 1, deg, 0, 0, 1, 0, 0, 0, 0 };
```
Comment (JSUYA) [inline src/loaders/svg/tvgSvgLoader.cpp:792]:
```suggestion
            auto deg = tanf(points[0] * (M_PI / 180.0));
            Matrix tmp = { 1, deg, 0, 0, 1, 0, 0, 0, 1 };
```

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 2]
```diff
@@ -786,6 +786,16 @@ static Matrix* _parseTransformationMatrix(const char* value)
...
+            _matrixCompose(matrix, &tmp, matrix);
+        } else if (state == MatrixState::SkewY) {
+            if (ptCount != 1) goto error;
+            auto deg = (float)tan(points[0] * (M_PI / 180.0));
+            Matrix tmp = { 1, 0, 0, deg, 1, 0, 0, 0, 0 };
```
Comment (JSUYA) [inline src/loaders/svg/tvgSvgLoader.cpp:797]:
```suggestion
            auto deg = tanf(points[0] * (M_PI / 180.0));
            Matrix tmp = { 1, 0, 0, deg, 1, 0, 0, 0, 1 };
```

**Review by JSUYA — COMMENTED** [review]
Hi I checked this PR in SVG below.
It works well after applying the small modifications I left. :)

```html
<svg
  viewBox="-40 0 150 100"
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <g
    fill="grey"
    transform="rotate(-10 50 100)
               translate(-36 45.5)
               skewX(40)
               scale(1 0.5)">
    <path
      id="heart"
      d="M 10,30 A 20,20 0,0,1 50,30 A 20,20 0,0,1 90,30 Q 90,60 50,90 Q 10,60 10,30 z" />
  </g>

  <use href="#heart" fill="none" stroke="red" />
</svg>

```

**Comment by MewPurPur** [comment]
OK thanks, I've added the tweaks

**Review by JSUYA — APPROVED** [review]
LGTM :)
+) Could you please edit commit message to make it look better? :)
https://github.com/thorvg/thorvg/blob/main/CONTRIBUTING.md

**Comment by hermet** [comment]
@MewPurPur thanks, that would be better if you add an example svg file in the thorvg/src/examples/resources.

**Pull request merged**